inc/Compatibility/Compatibility.php

```diff
@@ -17,7 +17,7 @@ public static function admin_notices(): void {
 		if ( ! self::is_gutenberg_plugin_active() ) {
 			wp_admin_notice(
 				__(
-					'The Gutenberg plugin has not been installed. The VIP Real-Time Collaboration plugin has been disabled.',
+					'The correct Gutenberg plugin has not been installed. The VIP Real-Time Collaboration plugin has been disabled.',
```

Comment:
I needed to communicate that the problem is that, our custom GB plugin has not been installed. Hence why I used correct here, and also avoided saying fork or VIP.